packages/api/src/Database/seeders/Seeders.service.ts

```diff
@@ -38,7 +39,7 @@ export class SeederService {
         firstName,
         lastName,
         email: `user${i}@test.com`,
-        password: `user${i}123`
+        password: await bcrypt.hash(`user${i}123`, 10)
```

Comment:
This should use the `UserService` a per https://github.com/code-mentoring/learn/issues/66